src/BaGet.Core/Authentication/BasicAuthenticationHeaderBuilder.cs

```diff
@@ -0,0 +1,13 @@
+using System.Net;
+using System.Net.Http.Headers;
+
+namespace BaGet.Core.Authentication
+{
+    public class BasicAuthenticationHeaderBuilder
+    {
+        public static AuthenticationHeaderValue CreateWith(NetworkCredential credential)
+        {
+            return new System.Net.Http.Headers.AuthenticationHeaderValue(BasicAuthenticationDefaults.AuthenticationScheme, AuthenticationHeaderValueExtensions.GetBase64BasicAuthHeaderString(credential));
```

Comment:
Do you need the full namespace here? Can this be simplified to just `AuthenticationHeaderValue`?